pkgs/by-name/li/libvpl/opengl-driver-lib.patch

```diff
@@ -0,0 +1,19 @@
+--- a/libvpl/src/mfx_dispatcher_vpl_loader.cpp
++++ b/libvpl/src/mfx_dispatcher_vpl_loader.cpp
+@@ -548,6 +548,16 @@ mfxStatus LoaderCtxVPL::BuildListOfCandidateLibs() {
+         it++;
+     }
+
++    // fourth priority: /run/opengl-driver/lib/
++    searchDirList.clear();
++    searchDirList.push_back("/run/opengl-driver/lib/");
```

Comment:
`@driverLink@` and `substituteAll { src = ...; inherit (addDriverRunpath) driverLink; }`